frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1643,6 +1636,18 @@ With this enabled, the current page is included, so the count goes from n to 1 i
                     self:refreshFooter(true, true)
                 end,
             },
+            {
+                text = _("Chapter progress"),
```

Comment:
May be this could be made more explicite? `[ ] Show only current chapter` or something?
And if not explicite, it may need a help_text.